packages/flutter/lib/src/widgets/implicit_animations.dart

```diff
@@ -1671,6 +1674,131 @@ class _AnimatedRotationState extends ImplicitlyAnimatedWidgetState<AnimatedRotat
   }
 }
 
+/// Widget which automatically transitions the child's
+/// offset relative to its normal position whenever the given offset changes.
+///
+/// The translation is expressed as an [Offset] scaled to the child's size. For
+/// example, an [Offset] with a `dx` of 0.25 will result in a horizontal
+/// translation of one quarter the width of the child.
+///
+/// {@tool snippet}
```

Comment:
Can this be an interactive `{@tool dartpad ...}` instead?

Reply:
Thanks, I changed it now